src/shims/unix/linux/fd/event.rs

```diff
@@ -59,11 +61,17 @@ impl FileDescriptor for Event {
         bytes: &[u8],
```

Comment:
There's a FIXME a few lines up, what's up with that?

I assume this should use the endianess of the target. Currently you are using `from_be_bytes`. That is strange since x86 is little-endian?

Reply:
I confused myself on the order of the bytes. These bytes are written in native endianess (see an example usage) https://github.com/tokio-rs/mio/blob/master/src/sys/unix/waker.rs#L31

Reply:
> These bytes are written in native endianess 

All right, that's what I expected. So this code should match on the target endianess (in `sess.target.endian`), and call `from_be_bytes`/`from_le_bytes` as appropriate. Might make sense to have a helper function `from_target_endianess_bytes`.

The annoying part is that we already have exactly this logic in `Allocation::read_scalar`. But here we get the already-read buffer of `u8`... if we got the original pointer to the user-provided buffer instead, we could just do `read_scalar`... well we'd also have to reset the alignment requirement to 1... not sure if that would be any better in the end.

Reply:
Yeah let's stick with the current approach for now.